### PR TITLE
Update MediaWiki to 1.29.1

### PR DESCRIPTION
**Pull request by davidbarratt**

### CODE REVIEW

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 6cde0f6..6d59a71 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,3 +1,4 @@
 mediawiki:1.27.3 @ amd64
 mediawiki:1.28.2 @ amd64
 mediawiki:1.29.0 @ amd64
+mediawiki:1.29.1 @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index cf40d84..01288d0 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -4,6 +4,7 @@ mediawiki:1.28
 mediawiki:1.28.2
 mediawiki:1.29
 mediawiki:1.29.0
+mediawiki:1.29.1
 mediawiki:latest
 mediawiki:legacy
 mediawiki:lts
diff --git a/mediawiki_1.29.0/Dockerfile b/mediawiki_1.29.1/Dockerfile
similarity index 88%
copy from mediawiki_1.29.0/Dockerfile
copy to mediawiki_1.29.1/Dockerfile
index c688a6e..3936e0e 100644
--- a/mediawiki_1.29.0/Dockerfile
+++ b/mediawiki_1.29.1/Dockerfile
@@ -33,8 +33,8 @@ RUN mkdir -p /var/www/data \
 # Version
 ENV MEDIAWIKI_MAJOR_VERSION 1.29
 ENV MEDIAWIKI_BRANCH REL1_29
-ENV MEDIAWIKI_VERSION 1.29.0
-ENV MEDIAWIKI_SHA512 3b77d04f13488b2a602d9030a4833c881188c93e2708ea7941d9e82f7a7215f3028695f396914da09ae1288cb9e980cc04f78559edaff6559e3d50b0c47afeb3
+ENV MEDIAWIKI_VERSION 1.29.1
+ENV MEDIAWIKI_SHA512 c4e04c4fb665c3d8299f3e03e608904aaf0e06381240c7259813eb670c3e32cde919353dd19993250cf49be81d604ac5f6d468bc563116a4b268e5011d34119f
 
 # MediaWiki setup
 RUN curl -fSL "https://releases.wikimedia.org/mediawiki/${MEDIAWIKI_MAJOR_VERSION}/mediawiki-${MEDIAWIKI_VERSION}.tar.gz" -o mediawiki.tar.gz \
````

</details>

File: library/mediawiki [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-GitCommit: 76ad8cc3ddc56701522025583c126ad38335a364
+GitCommit: 659cf5aedb8f785bb400a63f9c450f458ead260c
+
+Tags: stable, latest, 1.29, 1.29.1
+Directory: stable
 
-Tags: stable, latest, 1.29, 1.29.0
+Tags: 1.29.0
+GitCommit: 76ad8cc3ddc56701522025583c126ad38335a364
 Directory: stable
```
Comment (tianon) [inline library/mediawiki]: Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Comment (davidbarratt) [reply library/mediawiki]: @tianon oh! ok, yeah no reason to keep it. It is no longer supported.

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index 6cde0f6..22e2500 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,3 +1,3 @@
 mediawiki:1.27.3 @ amd64
 mediawiki:1.28.2 @ amd64
-mediawiki:1.29.0 @ amd64
+mediawiki:1.29.1 @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index cf40d84..a6ffb85 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -3,7 +3,7 @@ mediawiki:1.27.3
 mediawiki:1.28
 mediawiki:1.28.2
 mediawiki:1.29
-mediawiki:1.29.0
+mediawiki:1.29.1
 mediawiki:latest
 mediawiki:legacy
 mediawiki:lts
diff --git a/mediawiki_1.29.0/Dockerfile b/mediawiki_1.29.1/Dockerfile
similarity index 88%
rename from mediawiki_1.29.0/Dockerfile
rename to mediawiki_1.29.1/Dockerfile
index c688a6e..3936e0e 100644
--- a/mediawiki_1.29.0/Dockerfile
+++ b/mediawiki_1.29.1/Dockerfile
@@ -33,8 +33,8 @@ RUN mkdir -p /var/www/data \
 # Version
 ENV MEDIAWIKI_MAJOR_VERSION 1.29
 ENV MEDIAWIKI_BRANCH REL1_29
-ENV MEDIAWIKI_VERSION 1.29.0
-ENV MEDIAWIKI_SHA512 3b77d04f13488b2a602d9030a4833c881188c93e2708ea7941d9e82f7a7215f3028695f396914da09ae1288cb9e980cc04f78559edaff6559e3d50b0c47afeb3
+ENV MEDIAWIKI_VERSION 1.29.1
+ENV MEDIAWIKI_SHA512 c4e04c4fb665c3d8299f3e03e608904aaf0e06381240c7259813eb670c3e32cde919353dd19993250cf49be81d604ac5f6d468bc563116a4b268e5011d34119f
 
 # MediaWiki setup
 RUN curl -fSL "https://releases.wikimedia.org/mediawiki/${MEDIAWIKI_MAJOR_VERSION}/mediawiki-${MEDIAWIKI_VERSION}.tar.gz" -o mediawiki.tar.gz \
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #3366; 9f4c5309 (`mediawiki`):

```console

$ bashbrew build mediawiki:stable
Building bashbrew/cache:d4f3309b98b52269793a8d69247b05d66ad7b288a88a1f2128c2745f9d7b00e7 (mediawiki:stable)
Tagging mediawiki:stable
Tagging mediawiki:latest
Tagging mediawiki:1.29
Tagging mediawiki:1.29.1

$ test/run.sh mediawiki:stable
testing mediawiki:stable
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build mediawiki:legacy
Building bashbrew/cache:5d5baf5d7a5009cb7c7f143e46712cfdfe1747b97a3c05a1e09c0eb492a9d1f7 (mediawiki:legacy)
Tagging mediawiki:legacy
Tagging mediawiki:1.28
Tagging mediawiki:1.28.2

$ test/run.sh mediawiki:legacy
testing mediawiki:legacy
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build mediawiki:lts
Building bashbrew/cache:602ef4f2eeff138fafb28e9084679b5f865314c672dec2c3c8ab01262a64153c (mediawiki:lts)
Tagging mediawiki:lts
Tagging mediawiki:1.27
Tagging mediawiki:1.27.3

$ test/run.sh mediawiki:lts
testing mediawiki:lts
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**